exercise6.rb

```diff
@@ -0,0 +1,20 @@
+# 6. Write a function to find the maximum element in an array.
+
+def find_max_element(arr)
+  max_element = arr[0]
+
+  for element in arr[1..-1]
```

Comment:
Please use the range operator here. 
https://www.studytonight.com/ruby/ranges-in-ruby

Reply:
(arr[1..-1]).each do |element|
  max_element = element if max_element.nil? || element > max_element
end